release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2779,6 +2779,64 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.15.49
+[id="ocp-4-15-49_{context}"]
+=== RHSA-2025:3790 - {product-title} 4.15.49 bug fix and security update
+
+Issued: 16 April 2025
+
+{product-title} release 4.15.49, which includes security updates, is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2025:3790 [RHSA-2025:3790] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHBA-2025:3792[RHBA-2025:3792] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.49 --pullspecs
+----
+
+[id="ocp-4-15-49-known-issues_{context}"]
+==== Known issues
+
+* IPsec is not supported on {op-system-base-full} compute nodes because of a `libreswan` incompatiblility issue between a host and an `ovn-ipsec` container that exist in each compute node.  (link:https://issues.redhat.com/browse/OCPBUGS-36688[*OCPBUGS-36688*]) 
+
+[id="ocp-4-15-49-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, an update to the {ibm-cloud-name} Cloud Internet Services (CIS) implementation impacted the upstream Terraform plugin. If you attempted to create an external-facing cluster on {ibm-cloud-name}, the following error occurred:
++
+[source,terminal]
+----
+ERROR Error: Plugin did not respond
+ERROR
+ERROR with module.cis.ibm_cis_dns_record.kubernetes_api_internal[0],
+ERROR on cis/main.tf line 27, in resource "ibm_cis_dns_record" "kubernetes_api_internal":
+ERROR 27: resource "ibm_cis_dns_record" "kubernetes_api_internal"
+----
++
+With this release, you can use the installation program to create an external cluster on {product-title} without the plugin issue. (link:https://issues.redhat.com/browse/OCPBUGS-54367[OCPBUGS-54367]) 
+
+// Awaiting approval from Marco Braga for OCPBUGS-54353
```

Comment:
Are the bugs with "Awaiting approval" approved? If they are approved, remove the comments. If they aren't is it too early to merge?